config/sync/node.type.magazine.yml

```diff
@@ -9,7 +9,7 @@ third_party_settings:
     available_menus:
       - main
     parent: 'main:'
-name: Magazine
+name: Magasin
```

Comment:
I think this should be left as `Magazine` and translated to Danish.
We may have a mess with languages since we have content type translations for both Danish and English.